exporters/committime/collector_base.py

```diff
@@ -174,6 +174,10 @@ def get_metric_from_build(self, build, app, namespace, repo_url):
         errors = []
         try:
             metric = commit_metric_from_build(app, build, errors)
+
+            if not self._is_metric_ready(namespace, metric, build):
+                return None
```

Comment:
Should there be at least warning/debug that the metric isn't yet available?

Reply:
The logging happens in `_is_metric_ready` because it can offer greater perspective as to why it was skipped.

Reply:
Ok great.